layouts/_default/single.html

```diff
@@ -6,9 +6,9 @@ <h1 class="post-title">
   <div class="post-meta">
     {{- if .Date -}}
       <time class="post-date">
-        {{- .Date.Format "2006-01-02" -}}
-        {{- if $.Site.Params.showLastUpdated -}}
-          [{{- or $.Site.Params.updatedDatePrefix "Updated" -}} :: {{- .Lastmod.Format "2006-01-02" -}}]
```

Comment:
@KatieTheDev Looks good and works well 💪 But I found that you removed the `[{{- or $.Site.Params.updatedDatePrefix "Updated" -}} :: ...` string from the brackets. Was it intentional?